tests/phpunit/CRM/Contact/Form/ContactTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+ /**
+  * @group headless
+  */
+class CRM_Contact_Form_ContactTest extends CiviUnitTestCase {
+
+  /**
+   * Contact ID.
+   *
+   * @var int
+   */
+  protected $_individualId;
```

Comment:
can we use `$individualID` (I know there are a few like this already but I do slowly change them, and add the `:void` to the functions

Reply:
That was copypasta.  I also don't know how to write ajax tests.